schemas/controller-extension.yaml

```diff
@@ -12,7 +12,6 @@ paths:
           in: query
           description: base-64 encoded data
           required: false
-          style: form
```

Comment:
This was technically not valid here; it didn't cause a problem, but I'd rather not have ignored things in the basic examples.